src/Service/InvitePartnerService.php

```diff
@@ -194,4 +216,99 @@ public function invitePartnerHandler(?array $data=[], ?array $configuration=[]):
     }//end invitePartnerHandler()
 
 
+    /**
+     * This function creates a person object for the given BRP person.
+     *
+     * @TODO: Probably we want to move this to a mapping.
+     *
+     * @param array        $huwelijk  The marriage array given by the frontend.
+     * @param ObjectEntity $brpPerson The person from the BRP.
+     *
+     * @return ObjectEntity The person in the contact.
+     */
+    private function createPerson(array $huwelijk, ?ObjectEntity $brpPerson=null): ?ObjectEntity
```

Comment:
haven't you added this function to the assentService?

Reply:
I did, I'll remove it here